docs/notebooks/feasible_sets.pct.py

```diff
@@ -230,6 +240,77 @@ def objective_function(x):
 )
 
 
+# %% [markdown]
+# ## Active learning with Integrated Variance Reduction acquisition function
+#
+# An alternative to the `ExpectedFeasibility` function is called `IntegratedVarianceReduction`. This acquisition has the advantage of allowing batches, as we illustrate below. It is more expensive to compute than `ExpectedFeasibility`, as it writes as an integral over a set of input locations.
```

Comment:
```suggestion
# An alternative to the `ExpectedFeasibility` acquisition function is called `IntegratedVarianceReduction`. This acquisition has the advantage of taking into account reduction of uncertainty in a region of the search space when choosing the next point to sample, instead of considering only the sampling point. This makes it more expensive to compute than `ExpectedFeasibility`, since it involves computing an integral over a set of integration points. This integration region is determined by the user, with the `integration_points` parameter. `IntegratedVarianceReduction` also allows batches, as we illustrate below.
```